src/xdotoolify.js

```diff
@@ -298,10 +331,17 @@ _Xdotoolify.prototype.do = async function() {
         commandArr = [];
         await _sleep(op.ms);
       } else if (op.type === 'mousemove') {
+        await this._do(commandArr.join(' '));
+        commandArr = [];
+
         var pos = op.selector;
         if (typeof op.selector === 'string' || Array.isArray(op.selector)) {
+          let timeout = op.timeout || this.defaultTimeout;
+          if (timeout) {
+            await waitUntilElementIsAvailable(this.page, op.selector, timeout);
+          }
           var ret = await _getElementAndBrowserScreenRect(
-            this.page, op.selector
+            this.page, op.selector, op.timeout || 10000
```

Comment:
is this meant to be 10000 or defaultTimeout?

Reply:
or rather, should this line not accept a timeout at all since it waits just before?

Reply:
Good catch